modules/orionode/lib/args.js

```diff
@@ -45,7 +45,9 @@ exports.createDirs = function(dirs, callback) {
 exports.readPasswordFile = function(passwordFile) {
 	if (passwordFile) {
 		try {
-			return fs.readFileSync(passwordFile,'utf8');
+			var lines = fs.readFileSync(passwordFile,'utf8')
+				.split('\n').filter(Boolean);
+			return lines.length>0?lines[0]:null;
```

Comment:
My comments are:
1) Is it by convention only the first line of a passwordFile treated as the password? rather then the whole content(with out carriage in between), so I don't know if this is the right way by design?
2) Why filter(Boolean), I can't see what this code is doing. Array.filter expects a callback, but I don't know what a Boolean Constructor is doing here.